test/test_namedtuple_return_api.py

```diff
@@ -27,6 +27,10 @@
 
 class TestNamedTupleAPI(TestCase):
 
+    def test_import_return_types(self):
+        import torch.return_types  # noqa: F401
+        exec('from torch.return_types import *')
```

Comment:
There was an [issue](https://github.com/pytorch/pytorch/issues/62793) with `from torch import *` failing. I thought it might be good to have a case similar test case for this.

Reply:
I'm not a big fan of the exec but maybe its fine?
Maybe we can have a separate test file that just import * everything?

Reply:
We already have one. I think it would be ok to use exec here as we are not passing any arbitrary string.

https://github.com/pytorch/pytorch/blob/a6c0edff1a60e299d3f9794407f0a1c9a269fd19/test/test_torch.py#L129-L130

Reply:
If Alban is OK with having this test, then we should add a comment about what this is testing. I'm neutral on this test; `from blah import *` is not something you'd expect to fail haha